puppet/data/common.yaml

```diff
@@ -20,6 +20,7 @@ profiles::backup::receiver::targets:
   - redmine01
   - master02
```

Comment:
Slightly related question: shouldn't this be `controller01` now? I may have messed that up in the Jenkins migration.

Reply:
Most probably it should